src/c++/perf_analyzer/data_loader.cc

```diff
@@ -121,6 +139,17 @@ DataLoader::ReadDataFromDir(
       SerializeStringTensor(output_string_data, &it->second);
     }
   }
+
+  for (const auto& file : std::filesystem::directory_iterator(data_directory)) {
```

Comment:
Thanks for catching this! I tried a lot of different approaches and must have missed this during clean-up. I'll remove it.

Can you explain the output validation file part? This may be something new to me.

Reply:
Oh, I think I understand. This only covers inputs. One second, I'll push a commit to cover outputs as well.

Reply:
Removed the duplicate and added outputs to the I/O which is checked. It builds and looks to have the same test failures.